### PR TITLE
[10.0][ADD] analytic_account_sequence

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Generate an automatic and unique code for the analytic hierarchy

### CODE REVIEW

File: analytic_account_sequence/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    'name': 'Analytic account code sequence',
+    'summary': 'Analytic account code sequence',
+    'version': '10.0.1.0.0',
+    'author':   'Eficent, '
+                'SerpentCS ,'
+                'Project Expert Team ,'
+                'Odoo Community Association (OCA)',
+    'website': 'https://www.github.com/OCA/account-analytic',
+    'category': 'Project Management',
+    'depends': ['analytic', 'project', 'account_analytic_parent'],
```
Comment (pedrobaeza) [inline analytic_account_sequence/__manifest__.py]: project?

File: analytic_account_sequence/i18n/analytic_account_sequence.pot [hunk 2]
```diff
@@ -0,0 +1,266 @@
+# Translation of Odoo Server.
```
Comment (pedrobaeza) [inline analytic_account_sequence/i18n/analytic_account_sequence.pot]: Remove this

File: analytic_account_sequence/__manifest__.py (hunk 1 above)
Comment (AaronHForgeFlow) [reply analytic_account_sequence/__manifest__.py]: It depends but it should not, I'll remove all the dependent code.

**Comment by feketemihai** [comment]
@aheficent @pedrobaeza the module is merged in 10 and much simplified, since it uses ir. sequence... I think much simpler to maintain it.. I think this one should be closed..

**Comment by pedrobaeza** [comment]
Yeah, I agree. I have written in the other PR because I remembered that @aheficent was working in something similar.

**Pull request closed without merging**